telco-ran/configuration/kube-compare-reference/metadata.yaml

```diff
@@ -115,7 +115,10 @@ parts:
       - name: node-tuning-operator
         allOf:
           - path: node-tuning-operator/PerformanceProfile.yaml
+      - name: optional node-tuning-operator
+        anyOf:
           - path: node-tuning-operator/TunedPerformancePatch.yaml
+          - path: node-tuning-operator/TunedPowerCustom.yaml
```

Comment:
Almost - We want the main `TuningPerformancePatch.yaml` in the `allOf` section, and only the optional `TunedPowerCustom.yaml` in the `anyOf` section.

Reply:
Done.